workers/unity/Assets/Fps/Scripts/SetupLogic/OneTimeInitialisation.cs

```diff
@@ -23,6 +23,7 @@ private static void Init()
 
             // Setup template to use for player on connecting client
             PlayerLifecycleConfig.CreatePlayerEntityTemplate = FpsEntityTemplates.Player;
+            PlayerLifecycleConfig.MaxNumFailedPlayerHeartbeats = 5;
```

Comment:
increased that number to reduce the change of this heartbeating failing first.
but open to revert that change if people think that's not necessary